build/commands/scripts/sync.js

```diff
@@ -19,16 +20,23 @@ program
   .option('--target_android_base <target_android_base>', 'target Android OS level for apk or aab (classic, modern, mono)')
   .option('--init', 'initialize all dependencies')
   .option('--force', 'force reset all projects to origin/ref')
-  .option('--ignore_chromium', 'do not update chromium version even if it is stale')
+  .option('--sync_src [arg]', 'force or skip src sync (true/false/1/0)', JSON.parse)
```

Comment:
This flag existed to prevent update (and cause reset and longer build) but we still want to run sync and apply patches which could have changed.

Reply:
> This flag existed to prevent update (and cause reset and longer build) but we still want to run sync and apply patches which could have changed.

with `--src_sync=false`, patch apply and brave sync will still work as usual:
```
  const didSyncChromium = syncChromium(program)
  if (!didSyncChromium) {
    // If no Chromium sync was done, run sync inside `brave` to sync Brave DEPS.
    syncBrave(program) // <<< this will be called
  }
  Log.progress('...gclient sync done.')

  await util.applyPatches() // <<< this also will be called no matter --src_sync value
```

Reply:
Is there a reason we can't also keep the old option?

Reply:
How about we meet in the middle and call it `--sync_chromium=always|false`? I like your new param but users may still view src as being both chromium _and_ brave (since brave is still in src/), so keeping it referred as chromium in the flags may help it be clear. We can also keep --ignore_chromium (as deprecated) or at least log an error about the changed param name.